main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -1558,7 +1558,7 @@ object Lowering {
         val loc = e.loc.asSynthetic
         val e1 = mkChannelExp(sym, e.tpe, loc) // The channel `ch`
         val e2 = mkPutChannel(e1, e, Type.Impure, Type.mkUnion(e.eff, e1.eff, loc), loc) // The put exp: `ch <- exp0`.
-        val e3 = LoweredAst.Expression.Spawn(e2, Type.Unit, Type.Impure, e2.eff, loc) // Spawn the put expression from above i.e. `spawn ch <- exp0`.
+        val e3 = LoweredAst.Expression.Spawn(e2, LoweredAst.Expression.Region(Type.Unit, loc), Type.Unit, Type.Impure, e2.eff, loc) // Spawn the put expression from above i.e. `spawn ch <- exp0`.
```

Comment:
This line will later become a problem, I suspect.

Reply:
Yeah, it's basically duplicating `@ Static`. But given that we already have lots of `@ Static` which we'll have to cope with, it struck me that this wasn't a problem for now.

Reply:
Yes, let us just try to remind each other.

Reply:
I don't think that will be a problem: we're going to have to give Flix a `Static`-ectomy at some point, and this will fall out in the wash when we do so I suspect.